plans/dht/metrics/metrics.go

```diff
@@ -0,0 +1,29 @@
+package metrics
+
+import (
+	"os"
+	"time"
+
+	"github.com/ethereum/go-ethereum/metrics"
+	"github.com/ethereum/go-ethereum/metrics/influxdb"
+)
+
+var (
+	endpoint  = "http://influxdb:8086"
+	database  = "metrics"
+	username  = ""
+	password  = ""
+	namespace = ""
+	tags      = map[string]string{}
+)
+
+func Setup() {
```

Comment:
Initialisation for the InfluxDB reporter, with sane defaults that could work out-of-the-box in our cluster env.